Marlin/Configuration.h

```diff
@@ -215,7 +215,7 @@ Here are some standard links for getting your machine calibrated:
 // If your configuration is significantly different than this and you don't understand the issues involved, you probably
 // shouldn't use bed PID until someone else verifies your hardware works.
 // If this is enabled, find your own PID constants below.
-//#define PIDTEMPBED
+#define PIDTEMPBED
```

Comment:
Oups :) That musst have come in there by accident, of course it wouldn't make sense to override the default config.